core/meta/inc/TDictionary.h

```diff
@@ -167,15 +167,15 @@ namespace ROOT {
 class TDictionary : public TNamed {
 
 private:
-   TDictAttributeMap *fAttributeMap;    //pointer to a class attribute map
-   ULong64_t fUpdatingTransactionCount; //!the Cling ID of the transaction that last updated the object
+   TDictAttributeMap *fAttributeMap{nullptr};    //pointer to a class attribute map
+   ULong64_t fUpdatingTransactionCount{0}; //!the Cling ID of the transaction that last updated the object
 
 protected:
    Bool_t              UpdateInterpreterStateMarker();
 
 public:
-   TDictionary(): fAttributeMap(0), fUpdatingTransactionCount(0) { }
-   TDictionary(const char* name): TNamed(name, ""), fAttributeMap(0), fUpdatingTransactionCount(0) { }
+   TDictionary(): fAttributeMap(nullptr), fUpdatingTransactionCount(0) { }
```

Comment:
```suggestion
   TDictionary() = default;
```

Reply:
Philippe, it is not allowed to use `default` constructor for `TObject` - derived classes.
Here more details: https://sft.its.cern.ch/jira/browse/ROOT-10300

Reply:
right, I forgot :(